### PR TITLE
feat: add new commands, remove deprecated

### PR DESCRIPTION
**Pull request by forivall**
bugfixes with targets that have `:` in them

toplevel targets are also now completed.

I also have a whole host of tweaks at https://github.com/forivall/nx-completion/tree/cleanup, let me know if you'd be open to that as a PR too.

### CODE REVIEW

File: nx-completion.plugin.zsh [hunk 1]
```diff
@@ -330,20 +343,22 @@ _nx_command() {
...
-        "--skipNxCache[Rerun the tasks even when the results are available in the cache.]" \
-        "--configuration[This is the configuration to use when performing tasks on projects.]:configuration:" \
-        "--withDeps[TInclude dependencies of specified projects when computing what to run.]" \
-        "--onlyFailed[Isolate projects which previously failed.]" \
-        "--verbose[Print additional error stack trace on failure.]" && ret=0
+        "(--skip-nx-cache --skipNxCache)"{--skip-nx-cache,--skipNxCache}"[Rerun the tasks even when the results are available in the cache.]" \
+        "(-t --target --targets)"{-t=,--target=,--targets=}"[Task(s) to run for affected projects.]:target:" \
+        "--version[Show version number.]" \
+        "--verbose[Print additional error stack trace on failure.]" \
+        && ret=0
```
Comment (forivall) [inline nx-completion.plugin.zsh:359]: sorted for easier maintenance

**Comment by edbzn** [comment]
Thanks for the improvements @forivall! 

> I also have a whole host of tweaks at https://github.com/forivall/nx-completion/tree/cleanup, let me know if you'd be open to that as a PR too.

Go ahead, open it!

File: nx-completion.plugin.zsh [hunk 2]
```diff
@@ -330,20 +343,22 @@ _nx_command() {
...
     ;;
-    (run)
+    (run|run-one)
```
Comment (forivall) [inline nx-completion.plugin.zsh:361]: interestingly, this alias was removed from nx 15.7 -> 15.8 >_<

Comment (edbzn) [reply nx-completion.plugin.zsh:361]: Weird, this is not documented in their release notes.

**Pull request merged**